docs/jwst/charge_migration/description.rst

```diff
@@ -0,0 +1,52 @@
+Description
+===========
+
+:Class: `jwst.charge_migration.ChargeMigrationStep`
+:Alias: charge_migration
+
+
+Overview
+--------
+This step corrects for an artifact seen in undersampled NIRISS images that may depress flux 
+in resampled images. The artifact is seen in dithered images where the star is centered in 
+a pixel. When the peak pixels of such stars approach the saturation level, they suffer from 
+significant :ref:`charge migration <charge_migration>`:
```

Comment:
Is this a self-reference, i.e. a link right back to the step docs we're currently reading? If so, I think the reference/link should be removed and just have "charge migration" appear as plain text. It's referring to the physical process of charge migration, not the correction step in the pipeline.

Reply:
No, it is a link to the 'charge migration' section in the saturation step docs. The text could be modified, for example, to be  "... they suffer from significant charge migration as described in the docs for the saturation step; the spilling ..." (leaving the link on 'charge migration' ) to emphasize that it is not referring to the step.

Reply:
OK, if it's a reference to another doc somewhere that explains charge migration that's fine. Just leave as is.

Reply:
OK